public/js/newsManagerController.js

```diff
@@ -0,0 +1,39 @@
+(function(exports){
+
+function NewsManagerController (storySummary, headlines, NewsManagerModel, NewsManagerView, ApiManager) {
```

Comment:
Probably `NewsManagerModel`, `NewsManagerView` and `ApiManager` could begin with lowercase letters (eg `newsManagerModel`.  This would indicate they are instances, rather than constructor functions.